db/schema.rb

```diff
@@ -15,6 +15,20 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "floofs", id: :serial, force: :cascade do |t|
+    t.string "name", null: false
+    t.integer "floofiness"
+    t.string "image"
+  end
+
+  create_table "posts", id: :serial, force: :cascade do |t|
```

Comment:
what is a post? I would consider an alternative model name. Is this a `FloofReview`?

Reply:
I think that's an error. We shouldn't have any tables set up other than User. We'll delete that from our database with the next commit.